stable/coredns/templates/_helpers.tpl

```diff
@@ -53,9 +57,10 @@ Generate the list of ports automatically from the server definitions
             {{- end -}}
         {{- end -}}
 
-        {{/* If none of the zones specify scheme, default to UDP (CoreDNS defaults to dns://) */}}
+        {{/* If none of the zones specify scheme, default to both (CoreDNS defaults to dns:// on both tcp & udp) */}}
```

Comment:
If that's the case could you please update the comment above to say that `dns://` is both TCP and UDP?

Reply:
rephrased